coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -115,3 +140,23 @@ def get_ignore_comment(self, origin, language):
                         language=language))
                 ignore_comment = None
         return ignore_comment
+
+    def read_ignore_comments(self, original_line, comment_delimiter):
+        """
+        Reads Ignore Comment(s) already present in the line and
+        accordingly returns list of bears previously ignored
+        """
+
+        if comment_delimiter not in original_line:
+            return
+        else:
```

Comment:
as you do a `return` above, this `else` is useless, except it causes unnecessary indentation.